lib/ui/confirmorder/confirm_order_page.dart

```diff
@@ -0,0 +1,182 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:mskirana_app/models/order.dart';
+import 'package:mskirana_app/models/product.dart';
+import 'package:mskirana_app/ui/common/shared_bottom_bar.dart';
+import 'package:mskirana_app/ui/confirmorder/confirm_order_presenter.dart';
+import 'package:mskirana_app/ui/confirmorder/confirm_order_store_card.dart';
+import 'confirm_order_item_view.dart';
+
+final keyItem = new GlobalKey<ConfirmOrderItemViewState>();
+
+class ConfirmOrderPage extends StatefulWidget {
+  final List<dynamic> productsList;
+  ConfirmOrderPage({Key key, @required this.productsList}) : super(key: key);
+  @override
+  ConfirmOrderPageState createState() => ConfirmOrderPageState();
+}
+
+class ConfirmOrderPageState extends State<ConfirmOrderPage>
+    implements ConfirmOrderContract {
+  final GlobalKey<ScaffoldState> scaffoldkey = new GlobalKey<ScaffoldState>();
+  var callMeChkBox = true, reConfirmCheckBox = true;
+  int _addressOption = 0;
+
+  void saveOrder(context) {
+    List<Product> products = new List<Product>();
+    widget.productsList
+        .forEach((element) => {products.add(Product.fromJson(element))});
+    Order order = new Order();
+    order.products = products;
+    //Create oder api call
+    ConfirmOrderPresenter presenter;
```

Comment:
This looks like a local variable 🤔

Reply:
Added since inly one api call. Changed it to global, will get in next update.